go.mod

```diff
@@ -3,206 +3,203 @@ module sigs.k8s.io/node-feature-discovery
 go 1.22
 
 require (
+	cloud.google.com/go/compute v1.23.3
+	cloud.google.com/go/compute/metadata v0.2.3
+	github.com/Azure/azure-sdk-for-go v68.0.0+incompatible
```

Comment:
We shouldn't depend on this stuff directly. I don't believe any changes to go.mod would be needed in this PR